pkg/migrations/sql/preinstall/006-tables_ha.sql

```diff
@@ -14,6 +16,8 @@ CREATE TABLE SCHEMA_CATALOG.ha_leases_logs
     lease_until  TIMESTAMPTZ,          -- exclusive
     PRIMARY KEY (cluster_name, leader_name, lease_start)
 );
+GRANT SELECT ON TABLE SCHEMA_CATALOG.ha_leases_logs TO prom_reader;
+GRANT SELECT, INSERT, UPDATE, DELETE ON TABLE SCHEMA_CATALOG.ha_leases_logs TO prom_writer;
```

Comment:
Should this all be done with a function cause otherwise someone with writer perms could pretend to be writing when they're not, bringing down the system?

Reply:
But I guess if you're lying about that you could lie in the function too...

Reply:
(this could be done as the admin in a separate connection?)

Reply:
Yeah I don't think our security model is availability safe to nefarious prom_writers :)

Reply:
fair enough :)

Reply:
And I suppose the reader role is the one that you'd be handing out DB users in anyway. Fair enough.

Reply:
(Some folks do have a deleter role that's switched to for those things, and that would do this sort of stuff, but seems like overkill)